docs/templates/creating.md

```diff
@@ -0,0 +1,94 @@
+# Working with templates
+
+You create and edit Coder templates as [Terraform](./concepts.md) configuration
```

Comment:
Should be

`You create and edit Coder templates as [Terraform](./tour.md) configuration`

Reply:
Have we removed the concepts file?

Reply:
Yes. We replaced `concepts.md` with `tour.md`.